README.md

```diff
@@ -1 +1,87 @@
-# LF_test_task
+<h1> Test task service API </h1>
+
+<h3> REST API version </h3>
+The most recent version of the API is v1.
+
+<h2> Running Docker image <i>(Linux)</i></h2>
+
+In order to run this container you'll need docker [installed](https://docs.docker.com/install/)
+
+In the `project` directory run
+
+```shell
+$ sudo docker build -t my_docker_flask:latest .
+$ sudo docker run -d -p 5000:5000 my_docker_flask:latest
+```
+
+Check if container is running by going to [http://127.0.0.1:5000/](http://127.0.0.1:5000/)
+
+<h2> Available endpoints and how to use them </h2>
+
+<h3> Method 1: Get an average elapsed time for all queries on certain date </h3>
+
+<b>Description:</b> This endpoint returns an average elapsed time for all queries on certain date 
+
+<b>Request URL:</b> http://127.0.0.1:5000/api/v1/queries/avg_elapsed_time?date=[:date]
+
+<b>Request Values:</b> date in %Y-%m-%d format
+
+<b>Return Values:</b> Average elapsed time, sec
+
+<b>Example Request:</b> http://127.0.0.1:5000/api/v1/queries/avg_elapsed_time?date=2019-10-18
+
+<i> Estimated time:</i> 0.1 sec
```

Comment:
It is not clear how estimated time was measured. Usually it is measured multiple times, due to a lot of noise. Then you look into percentile metrics - p50, p95, p99 ... For example, p50 will tell you that 50% of users experience elapsed time less then that value.

Reply:
It was measured using _time_ module

Reply:
Imagine that question was asked by someone who knows anything about Python. And that person wants to understand what on average user will wait for the request to be completed.

Reply:
Ok I'll measure it